app/main.py

```diff
@@ -1,8 +1,59 @@
+from typing import Union
+
+
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 comfort_class: int, clean_mark: int, brand: str) -> None:
```

Comment:
It's better to use the same code style in all your code. So use indentation similar to the CarWashStation, also it has better readability

Reply:
> It's better to use the same code style in all your code. So use indentation similar to the CarWashStation, also it has better readability

So, is it ok, if my indentations throughout code will be like this?
![image](https://user-images.githubusercontent.com/107141441/217017097-02cf3097-f040-41c6-a798-b618c67d5ba6.png)